axelor-account/src/main/resources/domains/AccountConfig.xml

```diff
@@ -65,6 +65,8 @@
       title="Year opening account"/>
     <many-to-one name="yearClosureAccount" ref="com.axelor.apps.account.db.Account"
       title="Year closure account"/>
+    <integer name="accountCodeNbrChar" title="Account code nbr. char"
```

Comment:
accountCodeMinNbrChar --> accountCodeMinNbrCharSelect to follow the naming convention